docs/controllers/lidl-dimmer-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: controller
+title: Lidl Dimmer switch (Livarno)
+device: lidl-dimmer-switch
```

Comment:
Could you change the `device` to `HG06323`. Then, you will need to also change the name of the files from `lidl-dimmer-switch.{yml,md,jpeg}` to `HG06323.{yml,md,jpeg}`.